.github/workflows/container-publish.yaml

```diff
@@ -0,0 +1,98 @@
+name: Container image build and publish
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  schedule:
+    - cron: '45 17 * * *'
+  push:
+    branches: [ "master" ]
+    # Publish semver tags as releases.
+    tags: [ 'v*.*.*' ]
+  pull_request:
```

Comment:
Just out of curious, not knowing much about this workflow purpose actually...

Seeing some of main steps below are set to skipping on pull_request, do we need this workflow be triggered on `pull_request` event here?

Reply:
I've removed that stanza.